src/main/kotlin/no/nb/bikube/core/model/CollectionsObject.kt

```diff
@@ -16,6 +16,10 @@ data class CollectionsObject(
     @JsonProperty("@priref")
     val priRef: String,
 
+    val name: String? = null,
+
+    val term: String? = null,
+
```

Comment:
Should these be in a separate model? I know it will cause some duplicate/similar code, but might be confusing when we need to add more fields? Or is this easier to use? Need some discussion

Reply:
Creating separate model for language, publisher, publisherPlace